master.tex

```diff
@@ -57,7 +58,8 @@
 \usepackage{multirow}
 %\usepackage{booktabs}
 
-\usepackage{array}
+%\usepackage{array}
+\usepackage{array,longtable,color}
```

Comment:
Why do you add the longtable package? Do we have any tables spreading beyond a page break?

Reply:
sorry, was a leftover from my earlier tests. I will remove it